Formula/steem.rb

```diff
@@ -0,0 +1,39 @@
+class Steem < Formula
+  desc "Proof of Work blockchain with an unproven consensus algorithm."
+  homepage "https://steemit.com/"
+  url "https://github.com/steemit/steem.git",
+      :tag => "v0.12.1",
+      :revision => "0cca4bcd8212aed662a5aba4e35a5b75be3dcb1a"
+
+  depends_on :macos => [:yosemite, :el_capitan, :sierra]
+
+  depends_on "cmake" => :build
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on :python3 => :build
+  depends_on "doxygen" => :build
+  depends_on "berkeley-db" => :optional
+  depends_on "gperftools" => :optional
+  depends_on "qt5" => :optional
+  depends_on "icu4c"
+  depends_on "boost"
+  depends_on "openssl"
+  depends_on "readline"
+
+  def install
+    cmake_args = %W[
```

Comment:
This should be `= std_cmake_args + %W[` to avoid the `*(cmake_args + std_cmake_args)` at the bottotm.